zebra/main.c

```diff
@@ -365,6 +365,8 @@ int main(int argc, char **argv)
 			break;
 		case 's':
 			rcvbufsize = atoi(optarg);
+			if (rcvbufsize < 4194304)
+				flog_warn(EC_ZEBRA_NETLINK_EXTENDED_WARNING, "rcvbufsize is smaller than recommended value");
```

Comment:
This error code seems designed for warnings from Netlinks, but here it's just validation from arguments, right?